src/main/kotlin/frc/robot/commands/PickAndPlaceCommands.kt

```diff
@@ -0,0 +1,85 @@
+                          val elevatorSupplier: ()->Double,
+                          val elbowSupplier: ()->Double,
+                          val wristSupplier:()->Double,
+                          val intakeSupplier: ()->Double,
+                          ) : CommandBase() {
+
+    var elevatorPid = ProfiledPIDController(
+    ArmConstants.elevatorP,
+    ArmConstants.elevatorI,
+    ArmConstants.elevatorD, ArmConstants.elevatorTrapezoidConstraints, 0.02)
+
+    var elbowPid = ProfiledPIDController(
+        ArmConstants.elbowP,
+        ArmConstants.elbowI,
+        ArmConstants.elbowD, ArmConstants.elbowTrapezoidConstraints, 0.02)
+    var wristPid = ProfiledPIDController(
+        ArmConstants.wristP,
+        ArmConstants.wristI,
+        ArmConstants.wristD, ArmConstants.wristTrapezoidConstraints, 0.02)
+
+    val elbowFeedforward = ArmConstants.elbowFF
+
+    object Telemetry{
+        val desiredElbow = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("DesiredElbow").publish()
+        val desiredWrist = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("DesiredWrist").publish()
+        val desiredElevator = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("DesiredElevator").publish()
+        val desiredIntake = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("DesiredIntake").publish()
+
+        val elbowPidPosError = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElbowPidError").publish()
+        val elevatorPidPosError = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElevatorPidError").publish()
+        val wristPidPosError = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("WristPidError").publish()
+
+        val elbowPidVelocityError = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElbowPidError").publish()
+        val elevatorPidVelocityError = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElevatorPidError").publish()
+        val wristPidVelocityError = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("WristPidError").publish()
+
+        val elbowVoltage = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElbowVolts").publish()
+        val wristVoltage = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("WristVolts").publish()
+        val elevatorVoltage = NetworkTableInstance.getDefault().getTable("Arm").getDoubleTopic("ElevatorVolts").publish()
+    }
+
+    init{
+        elbowPid.setGoal(elbowSupplier())
+        elevatorPid.setGoal(elevatorSupplier())
+        wristPid.setGoal(wristSupplier())
+    }
+
+
+    override fun execute() {
+
+        subsystem.elevatorVoltage = elevatorPid.calculate(subsystem.elevatorPositionMeters,elevatorSupplier())
+        subsystem.elbowVoltage = elbowPid.calculate(subsystem.elbowEncoder.position,elbowSupplier())
```

Comment:
get rid of subsystem.elbowencoder.position and use elbowPositionRadians.
get rid of subsystem.wristencoder.position and use relativePitch or absolutePitch